### PR TITLE
Mark exported org.eclipse.swt.internal package as internal

### PR DESCRIPTION
**Pull request by iloveeclipse**
Fixes few API errors reported after 2ca0b37ecbeff105797d5fce72675bf306ad5282

### CODE REVIEW

File: bundles/org.eclipse.swt/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -15,7 +15,7 @@ Export-Package:
  org.eclipse.swt.dnd,
  org.eclipse.swt.events,
  org.eclipse.swt.graphics,
- org.eclipse.swt.internal,
+ org.eclipse.swt.internal;x-internal:=true,
```
Comment (tjwatson) [inline bundles/org.eclipse.swt/META-INF/MANIFEST.MF:18]: I think we still need `; x-friends:="org.eclipse.ui"` for to make ui a friend.  At least that is what the fragments do for this package.

Comment (HannesWell) [reply bundles/org.eclipse.swt/META-INF/MANIFEST.MF:18]: When searching `org.eclipse.ui` I can't see any usage of swt.internal anymore:
https://github.com/search?q=repo%3Aeclipse-platform%2Feclipse.platform.ui%20path%3A%2F%5Ebundles%5C%2Forg%5C.eclipse%5C.ui%5C%2F%2F%20org.eclipse.swt.internal&type=code

I already considered to remove it from the fragments too, but haven't done it yet.

Comment (iloveeclipse) [reply bundles/org.eclipse.swt/META-INF/MANIFEST.MF:18]: I don't see any warnings on `org.eclipse.ui` and looking on the code I don't think there is a need for that, it uses only public SWT type..

Do you see any warnings?

Comment (tjwatson) [reply bundles/org.eclipse.swt/META-INF/MANIFEST.MF:18]: Thanks for checking.  Then I agree, making x-internal is correct.

**Comment by HannesWell** [comment]
Thank you Andrey for this fix.

**Pull request merged**